docs/reference/commandline/build.md

```diff
@@ -277,7 +277,7 @@ flag](../run.md#specifying-custom-cgroups).
 
 Using the `--ulimit` option with `docker build` will cause each build step's
 container to be started using those [`--ulimit`
-flag values](../run.md#setting-ulimits-in-a-container).
+flag values](./run.md#set-ulimits-in-container-ulimit).
```

Comment:
Yes, I'm pretty sure `#set-ulimits-in-container-ulimit` is the right one, was just wondering why github is doing something else :stuck_out_tongue_closed_eyes: